website/_i18n/en.yml

```diff
@@ -193,7 +193,9 @@ dict:
       and you can run the Pactus blockchain on your personal computer with basic hardware.
     validators: What are validators?
     validators_answer: >
-      [version: 1] Validators on Pactus are responsible for processing transactions and creating blocks and, by doing so,
+      [version: 2]
+      Pactus Validator is software that you can run on your personal computer or a remote server.
```

Comment:
Pactus Validator is a software? Maybe rephrase?

Reply:
What is your suggestion?

Reply:
Pactus is a software that you can run

Reply:
Read the whole context:

```
Pactus Validator is a software that you can run on your personal computer or a remote server.
Validators on Pactus are responsible for processing transactions and creating blocks and, by doing so,
helping to secure the network and keep it decentralized.
They use their own coins as a stake to ensure the network's security and receive rewards for their contributions.
```

Reply:
Yeah i have read it There is no such software called "pactus validator" it is called Pactus.